### ISSUE TITLE
Memory-Leak in AcquireAlignedMemory MagickCore/memory.c:262

### ISSUE THREAD

**Issue by zhouat** [ImageMagick/ImageMagick#633]
Poc link : 
https://github.com/zhouat/poc_IM/blob/master/Memory-Leak-21_output_picon_1501391824.23

Trigger Command： `magick  convert Memory-Leak-21_output_picon_1501391824.23 out.picon`

```
=================================================================
==32246==ERROR: LeakSanitizer: detected memory leaks

...
...

Indirect leak of 512 byte(s) in 1 object(s) allocated from:
    #0 0x7f89eef2d076 in __interceptor_posix_memalign (/usr/lib/x86_64-linux-gnu/libasan.so.2+0x99076)
    #1 0x7f89ee671ab3 in AcquireAlignedMemory MagickCore/memory.c:262
    #2 0x7f89ee4b5c5b in OpenPixelCache MagickCore/cache.c:3575
    #3 0x7f89ee4ae43a in GetImagePixelCache MagickCore/cache.c:1682
    #4 0x7f89ee4bd4d7 in SyncImagePixelCache MagickCore/cache.c:5269
    #5 0x7f89ee63d86b in SetImageStorageClass MagickCore/image.c:2513
    #6 0x7f89ee4dc3af in AcquireImageColormap MagickCore/colormap.c:144
    #7 0x7f89ee842995 in ReadGIFImage coders/gif.c:1280
    #8 0x7f89ee516ed4 in ReadImage MagickCore/constitute.c:497
    #9 0x7f89ee49118b in BlobToImage MagickCore/blob.c:428
    #10 0x7f89ee97cf40 in WritePICONImage coders/xpm.c:689
    #11 0x7f89ee51a032 in WriteImage MagickCore/constitute.c:1114
    #12 0x7f89ee51acae in WriteImages MagickCore/constitute.c:1333
    #13 0x7f89ede735cf in ConvertImageCommand MagickWand/convert.c:3280
    #14 0x7f89edef370b in MagickCommandGenesis MagickWand/mogrify.c:183
    #15 0x4017e1 in MagickMain utilities/magick.c:149
    #16 0x4019c2 in main utilities/magick.c:180
    #17 0x7f89ed71f82f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2082f)




```

**Comment by mikayla-grace** [comment]
Unfortunately, we cannot reproduce this problem.  Its possible the fix for #631 also fixed this problem as well.

**Comment by bastien-roucaries** [comment]
CVE-2017-11754
